### PR TITLE
Added small test

### PR DESCRIPTION
**Pull request by Alrim42**
Summary of changes:
-
- updated glide.lock and glide.yaml
- added small test

How to verify it:
-
- run command `make test-small`

Testing done:
-
- `make` works 
- small test passed

### CODE REVIEW

File: threshold/threshold_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+package threshold
+
+import (
+	"testing"
+
+	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestChangeDetectorProcessor(t *testing.T) {
```
Comment (PatrykMatyjasek) [inline threshold/threshold_test.go]: this is not change detector processor

File: glide.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 package: github.com/intelsdi-x/snap-plugin-processor-threshold
 import:
-- package: github.com/golang/protobuf
```
Comment (PatrykMatyjasek) [inline glide.yaml]: why protobuf is removed?

File: threshold/threshold_test.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+				},
+			}
+
+			mts, err := proc.Process(metrics, config)
+			So(mts, ShouldNotBeNil)
+			So(err, ShouldBeNil)
+			So(mts, ShouldHaveLength, 2)
+		})
+
+		Convey("Process metrics with config containing static namespace", func() {
```
Comment (PatrykMatyjasek) [inline threshold/threshold_test.go]: please add same test, with value which should be filtered out

File: glide.yaml (hunk 2 above)
Comment (Alrim42) [inline glide.yaml]: probably because the plugin does not need it

File: threshold/threshold_test.go (hunk 3 above)
Comment (Alrim42) [inline threshold/threshold_test.go]: sure, I added this test in recent update of PR

**Pull request merged**